platform/src/main/java/org/stellar/anchor/platform/service/PaymentOperationToEventListener.java

```diff
@@ -169,10 +170,10 @@ public void onReceived(ObservedPayment payment) {
 
   @Override
   public void onSent(ObservedPayment payment) {
-    // noop
+    Log.info("NOOP PaymentOperationToEventListener#onSent was called.");
```

Comment:
I think it's better to use `Log.debug()` here.

Reply:
Updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/607/commits/247705aab4bb5e0d048fba7d6ae2d9c768103249.